packages/rocketchat-apps/client/communication/websockets.js

```diff
@@ -0,0 +1,34 @@
+export class AppWebsocketReceiver {
+	constructor(orch) {
+		this.orch = orch;
+		this.streamer = new Meteor.Streamer('apps');
+
+		this.streamer.on('app/added', this.onAppAdded.bind(this));
```

Comment:
if you define these as arrow function you won't have to do .bind(this) because the arrow functions don't create new scope.

```
onAppAdded = (appId) => {

}
```

Not really a change request.  Just could avoid having to do this unfortunate bind junk :)

Reply:
As that's a es7 feature and we are using eslint for es6, that style of development isn't possible. Also, the default scope of `this` in class functions is the class however the `Meteor.Streamer` messes with the scope and thus is what required the additional `.bind(this)` on setting the streamer callbacks.

Reply:
As per conversation -- due to something in meteor and/or the streamer we can use arrow functions there